WordPress/Classes/ViewRelated/Stats/Traffic/StatsTrafficDatePickerView.swift

```diff
@@ -0,0 +1,91 @@
+import SwiftUI
+import DesignSystem
+
+struct StatsTrafficDatePickerView: View {
+    @ObservedObject var viewModel: StatsTrafficDatePickerViewModel
+
+    var body: some View {
+        HStack {
+            Menu {
+                ForEach([StatsPeriodUnit.day, .week, .month, .year], id: \.self) { period in
+                    Button(period.label, action: {
+                        viewModel.period = period
+                    })
+                }
+            } label: {
+                Text(viewModel.period.label)
+                    .style(TextStyle.bodySmall(.emphasized))
+                    .foregroundColor(Color.DS.Foreground.primary)
+                Image(systemName: "chevron.down")
+                    .font(.system(size: 8))
+                    .foregroundColor(Color.DS.Foreground.secondary)
+
+            }
+            .menuStyle(.borderlessButton)
+            .padding(.vertical, Length.Padding.single)
+            .padding(.horizontal, Length.Padding.double)
+            .background(Color.DS.Background.secondary)
+            .clipShape(RoundedRectangle(cornerRadius: Length.Radius.max))
+            .overlay(
+                RoundedRectangle(cornerRadius: Length.Radius.max)
+                    .strokeBorder(.clear, lineWidth: 0)
+            )
+
+            Spacer()
+
+            Text(viewModel.formattedCurrentPeriod())
+                .style(TextStyle.bodySmall(.emphasized))
+                .foregroundColor(Color.DS.Foreground.primary)
+                .lineLimit(1)
+
+            Spacer().frame(width: Length.Padding.single)
+
+            Button(action: {
+                viewModel.goToPreviousPeriod()
+            }) {
+                Image(systemName: "chevron.left")
+                    .imageScale(.small)
+                    .foregroundColor(Color.DS.Foreground.secondary)
+                    .flipsForRightToLeftLayoutDirection(true)
+            }
+            .padding(.trailing, Length.Padding.single)
+
+            let isNextDisabled = !viewModel.isNextPeriodAvailable
+            let enabledColor = Color.DS.Foreground.secondary
+            let disabledColor = enabledColor.opacity(0.5)
+
+            Button(action: {
+                viewModel.goToNextPeriod()
+            }) {
+                Image(systemName: "chevron.right")
+                    .imageScale(.small)
+                    .foregroundColor(isNextDisabled ? disabledColor : enabledColor)
+                    .flipsForRightToLeftLayoutDirection(true)
+            }.disabled(isNextDisabled)
```

Comment:
nitpick, modifier alignment. Here and with `.padding(..)
```suggestion
            }
            .disabled(isNextDisabled)
```